.github/workflows/package.yml

```diff
@@ -22,8 +22,8 @@ jobs:
       fail-fast: false
       matrix:
         os: ${{
-          (github.event.inputs.os == 'all' && fromJSON('["ubuntu-latest", "macOS-latest", "windows-latest"]')) ||
-           (github.event.inputs.os == 'win' && fromJSON('["windows-latest"]')) ||
+          (github.event.inputs.os == 'all' && fromJSON('["ubuntu-latest", "macOS-latest", "windows-2019"]')) ||
+           (github.event.inputs.os == 'win' && fromJSON('["windows-2019"]')) ||
```

Comment:
Hmm... I'm curios what's special about windows-2019? I'm not even sure how Windows managers their versions by year

Reply:
Not fully sure, we just know that this stopped working on new windows runners. I'm not sure if this is still a problem for the newer honeycomb